src/test/java/com/visionit/TestClass2.java

```diff
@@ -2,6 +2,12 @@
 
 public class TestClass2 {
 
+    //Added by akash Done as a part of this ticket: jira-2386-add-search-test-cases/
+    public void test_00(){
```

Comment:
Change needed here on the line

Reply:
ok i will make the cahnges.

Reply:
changes are made, resolving the conversation.

Reply:
Ok Akash Sir, go for it